Gemfile

```diff
@@ -74,42 +73,43 @@ group :development do
   gem "guard-rspec", require: false
   gem "guard-rubocop", require: false
   gem "pry-rails"
-  gem "quiet_assets"
   gem "rack-mini-profiler", require: false
   gem "rails-erd"
+  gem "web-console"
 end
 
 group :development, :test do
-  gem "rspec-core", "~> 3.7.0"
+  gem "rspec-core"
   gem "rspec-rails"
 
   gem "awesome_print"
-  gem "binman", "~>5.1.0"
+  gem "binman"
   gem "brakeman", require: false
   gem "byebug"
   gem "database_cleaner"
-  gem "factory_girl_rails"
+  gem "factory_bot_rails"
   gem "ffaker"
   gem "grape-swagger-rails"
   gem "hirb"
-  gem "rubocop", "~> 0.53.0", require: false
-  gem "web-console", "~> 2.1.3"
+  gem "rubocop", require: false
   gem "wirb"
   gem "wirble"
 end
 
 group :test do
-  gem "capybara", "~> 2.14.3"
-  gem "capybara-screenshot", "~> 1.0.0"
+  gem "capybara"
+  gem "capybara-screenshot"
   gem "chromedriver-helper"
-  gem "codeclimate-test-reporter", require: false
-  gem "docker-api", "~> 1.28.0"
+  gem "docker-api"
   gem "json-schema"
-  gem "poltergeist", "~> 1.18.0", require: false
-  gem "selenium-webdriver", "~> 3.12"
+  gem "poltergeist", require: false
+  gem "rails-controller-testing"
+  gem "selenium-webdriver"
+  # TODO: I think it can be removed (shoulda-matchers does the trick already)
   gem "shoulda"
-  gem "simplecov", "0.15.1", require: false
+  gem "shoulda-matchers", require: false
```

Comment:
I think we should only keep `shoulda`. It already includes `shoulda-matchers` and `shoulda-context` based on its README.